Infrastructure.DataAccess/Migrations/SQLScripts/Migrate_To_Data_Processing_Registration.sql

```diff
@@ -0,0 +1,656 @@
+        SELECT
+			ItContractItSystemUsages.ItContractId, 
+			ItContractItSystemUsages.ItSystemUsageId,
+			ItContract.OrganizationId,
+			GETUTCDATE(),
+			'Konverteret_Databehandling_',
+			ItContract.ContainsDataHandlerAgreement,
+			ItContract.DataHandlerAgreementUrl,
+			ItContract.DataHandlerAgreementUrlName,
+			ItSystemUsage.dataProcessor,
+			CASE
+				WHEN ItSystemUsage.dataProcessorControl = 0 THEN 1
+				WHEN ItSystemUsage.dataProcessorControl = 1 THEN 0
+				WHEN ItSystemUsage.dataProcessorControl IS NULL THEN NULL
+				ELSE 2
+			END AS OversightConcluded,
+			ItSystemUsage.lastControl,
+			ItSystemUsage.noteUsage,
+			ItSystemUsage.datahandlerSupervisionDocumentationUrl,
+			ItSystemUsage.datahandlerSupervisionDocumentationUrlName,
+			ItContract.ObjectOwnerId,
+			ItContract.LastChangedByUserId,
+			ItSystemUsage.ObjectOwnerId
+        FROM 
+		ItContractItSystemUsages 
+			INNER JOIN ItContract ON ItContract.Id = ItContractItSystemUsages.ItContractId
+			INNER JOIN ItSystemUsage ON ItSystemUsage.Id = ItContractItSystemUsages.ItSystemUsageId
+		WHERE 
+			ItContractItSystemUsages.ItContractId 
+				IN (
+					SELECT 
+						Id 
+					FROM 
+						ItContract 
+					WHERE 
+						ContractTypeId = 5
+				)
+
+	DECLARE @DprIds1 table (rowNumber int IDENTITY(1,1) PRIMARY KEY, id int)
+
+	/*
+		Create new DPR's and save created Id's in a new table
+	*/
+
+	INSERT INTO 
+		DataProcessingRegistrations (OrganizationId, Name, LastChanged, IsAgreementConcluded, DataResponsibleRemark, IsOversightCompleted, LatestOversightDate, OversightCompletedRemark, ObjectOwnerId, LastChangedByUserId)
+	OUTPUT
+		inserted.Id into @DprIds1
+	SELECT
+		OrganizationId, Name, LastChanged, AgreementConcluded, DataResponsibleNote, OversightConcluded, OversightConcludedDate, OversighConcludedRemark, ContractUserId, ContractLastChangedBy
+	FROM 
+		@MigrationContext1 
+
+
+	DECLARE @DprsWithForeignKeys table (dprId int, ItContractId int, ItSystemUsageId int, AgreementConcludedUrlName varchar(max), AgreementConcludedUrl varchar(max), DatahandlerSupervisionLinkName varchar(max), DatahandlerSupervisionLink varchar(max), contractOwnerId int, systemOwnerId int)
+
+	/*
+		Create temp table with dprId's and Contract and SystemUsage Id's to be used when creating relations
+	*/
+
+	INSERT INTO 
+		@DprsWithForeignKeys (dprId, ItContractId, ItSystemUsageId, AgreementConcludedUrlName, AgreementConcludedUrl, DatahandlerSupervisionLinkName, DatahandlerSupervisionLink, contractOwnerId, systemOwnerId)
+	SELECT
+		id, ItContractId, ItSystemUsageId, AgreementConcludedUrlName, AgreementConcludedUrl, DatahandlerSupervisionLinkName, DatahandlerSupervisionLink, ContractUserId, SystemUsageUserId
+	FROM 
+		@MigrationContext1  as context
+		INNER JOIN @DprIds1 as dprs ON context.rowNumber = dprs.rowNumber
```

Comment:
Context tabellen med information der skal kobles på joines med DPR id's ud fra deres række nummer som bygger på hvornår dataen er indsat i de to tabeller.